### ISSUE TITLE
showyourwork fails if your local path has a space in it

### ISSUE THREAD

**Issue by bjmorgan** [showyourwork/showyourwork#257]
Running `showyourwork` from a top level git directory that has a space in the directory path fails:

```
msd-errors $ pwd
/Users/bjm42/Documents/Work/Writing/Papers/Active Papers/msd-errors
msd-errors $ showyourwork               
/bin/sh: Papers/msd-errors/.showyourwork/cache: No such file or directory
```

**Comment by dfm** [comment]
Brutal! I expect this has something to do with all the places where `subprocess` is called with `shell=True`. I've been wanting to refactor this to use [plumbum](https://plumbum.readthedocs.io), but in the meantime it would be useful to work out where exactly this is happening. Could you share the logs from the `.showyourwork/logs` directory if anything ends up in there?

**Comment by bjmorgan** [comment]
`.showyourwork/logs` is empty.

**Comment by dfm** [comment]
This should be fixed on `main` by #259. Thanks again for the report.
